### DISCUSSION TITLE
Logo for fooocus

### DISCUSSION THREAD

**Discussion by fenneishi** [lllyasviel/Fooocus#794] [Show and tell]
The logo for fooocus was created using fooocus.
![image (4)](https://github.com/lllyasviel/Fooocus/assets/33345538/1f8d6f20-9644-42a9-b233-7a7dc82d52f8)

**Comment by fenneishi** [comment]
I originally intended to create a logo for FooocusControl, but ended up creating a logo for fooocus.😅

**Reply by AkoZ** [reply]
here is a text not a logo ;) !
you need more graphical challenge and a bit squared due to screen format to pass to a logo.
but for a title it's the trick

**Reply by fenneishi** [reply]
Ha ha, you are right, it is a text and not a logo, I just share it for fun：)
However, I started to think about how to improve fooocus' ability to generate text, and I found that I really need this feature！
Do you have any suggestions or ideas

**Reply by kubilaykilinc** [reply]
I am graphic design teacher.
LOGO : it's mean LOGOTYPE - it is text based
Trademark or emblem is just symbol and sign based.

Nice works.

**Comment by barepixels** [comment]
About 3 weeks ago I made this for fun
![beepbeep](https://github.com/lllyasviel/Fooocus/assets/92839033/aca9731e-6adc-4d83-bf68-bd35cba77498)

**Reply by fenneishi** [reply]
cool !

**Comment by donlaiq** [comment]
A couple of ideas with the theme of dripping paint I've created using Fooocus
![2023-11-02_00-32-00_3219](https://github.com/lllyasviel/Fooocus/assets/10087643/72b618fc-9403-444b-b436-7207936b66ab)
![2023-11-02_00-37-57_1894](https://github.com/lllyasviel/Fooocus/assets/10087643/4342427a-3169-403f-bfdb-12b26bb7dc46)

**Reply by metercai** [reply]
what's the prompt?

**Reply by 3Diva** [reply]
@donlaiq  I really like the first one! Very nice!

**Reply by donlaiq** [reply]
The prompts are

![Captura desde 2023-11-12 09-11-26](https://github.com/lllyasviel/Fooocus/assets/10087643/3f1cfe28-b284-4389-855f-f0bb5d463e71)

and

![Captura desde 2023-11-12 09-12-02](https://github.com/lllyasviel/Fooocus/assets/10087643/4b1eaec8-eb39-4c6d-8a0e-f7c1264a291f)

Thanks, @3Diva ! :)
